app/page/template/content/preferences-account.htm

```diff
@@ -84,16 +84,17 @@
         <!-- /ko -->
       </section>
       <!-- ko if: isActivatedAccount() -->
-        <!-- ko if: z.util.Environment.desktop -->
-          <section class="preferences-section preferences-section-conversation-history">
-            <hr class="preferences-separator">
-            <header class="preferences-header" data-bind="l10n_text: z.string.preferencesOptionsBackupHeader"></header>
-            <div class="preferences-link text-theme" data-bind="click: clickOnBackupExport, l10n_text: z.string.preferencesOptionsBackupExportHeadline" data-uie-name="do-backup-export"></div>
-            <div class="preferences-detail" data-bind="l10n_text: z.string.preferencesOptionsBackupExportSecondary"></div>
-            <div class="preferences-link text-theme" data-bind="click: clickOnBackupImport, l10n_text: z.string.preferencesOptionsBackupImportHeadline" data-uie-name="do-backup-import"></div>
-            <div class="preferences-detail" data-bind="l10n_text: z.string.preferencesOptionsBackupImportSecondary"></div>
-          </section>
-        <!-- /ko -->
+        <section class="preferences-section preferences-section-conversation-history">
```

Comment:
I suppose we need some sort of feature check as this feature won't be supported on all platforms.

Reply:
Just for the record, this feature is going to be compatible with all the major browsers (the initial file streaming idea was a compatibility issue, but here it is not involved in the process)

Reply:
👍